e2e/smartvol_ops_test.go

```diff
@@ -316,6 +317,61 @@ func testSmartVolumeDistributeDisperse(t *testing.T) {
 	checkZeroLvs(r)
 }
 
+func editDevice(t *testing.T) {
+	r := require.New(t)
+	peerList, err := client.Peers()
+	r.Nil(err)
```

Comment:
if peerList len is zero fail the test case.

Reply:
I think this should be done in the test case of peerList and not in the edit-device.

Reply:
as this is dependent on the peer list, no harm in having one more higher level check